arch/arm/configs/nintendo3ds_defconfig

```diff
@@ -147,7 +147,7 @@ CONFIG_SCHED_AUTOGROUP=y
 # CONFIG_SYSFS_DEPRECATED is not set
 CONFIG_RELAY=y
 CONFIG_BLK_DEV_INITRD=y
-CONFIG_INITRAMFS_SOURCE="/home/wolfvak/linux-3ds/rootfs"
+# CONFIG_INITRAMFS_SOURCE is not set
```

Comment:
ah, so when I go in via menuconfig and remove this, it gets set to:

CONFIG_INITRAMFS_SOURCE=""

but I see the same panic you do with that.

I think we want

```diff
-CONFIG_INITRAMFS_SOURCE="/home/wolfvak/linux-3ds/rootfs"
+CONFIG_INITRAMFS_SOURCE="rootfs.cpio.gz"
```

Then that boots for me (though it seems that the bottom screen isn't initialized, and is missing the keyboard!)